conf/nginx.conf

```diff
@@ -0,0 +1,41 @@
+#sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
+location __PATH__/ {
+
+       # Path to source
+       alias __FINALPATH__/ ;
+
+       index index.php;
+
+      # if (!-e $request_filename)
+       #{
+       #       rewrite ^(.+)$ __PATH__/index.php?q=$1 last;
+       #}
+
+       if ($scheme = http) {
+             rewrite ^ https://$server_name$request_uri? permanent;
+       }
+
+       client_max_body_size 50m;
+
+       try_files $uri $uri/ index.php;
+       
+       location ~ [^/]\.php(/|$) {
+              fastcgi_split_path_info ^(.+?\.php)(/.*)$;
+              fastcgi_pass unix:/var/run/php/php__PHPVERSION__-fpm-__NAME__.sock;
+
+              fastcgi_index index.php;
+              include fastcgi_params;
+              fastcgi_param REMOTE_USER $remote_user;
+              fastcgi_param PATH_INFO $fastcgi_path_info;
+              fastcgi_param SCRIPT_FILENAME $request_filename;
+       }
+
+  # Include SSOWAT user panel.
+  include conf.d/yunohost_panel.conf.inc;
+}
+
+location ~ ^__PATH__/(data|config|\.ht|db_structure\.xml|README) {
+       deny all;
+}
+
+more_set_headers "X-Frame-Options : ALLOWALL";
```

Comment:
(I understand that this is copypasta of the old nginx.conf.j2, but I'm suspicious about why this needs to be on the global scope and not inside the location)